### DISCUSSION TITLE
Enable ccache with new remote database

### DISCUSSION THREAD

**Discussion by vaneyli** [ccache/ccache#357] [Ideas]
HI. I have wrote a new remote memory database instead of memcached for ccache.   Could I commit it?  
I have used it in my company for a long time, it works well.

**Comment by jrosdahl** [comment]
Sounds interesting, at least as input to a discussion about what a storage backend API (as mentioned briefly in comments to #58) should look like.

But to able to answer your question we would need to have at look at your implementation. Perhaps you could publish it somewhere?

**Comment by afbjorklund** [comment]
Start with https://github.com/ccache/ccache/blob/master/CONTRIBUTING.md and make a Pull Request ?

**Comment by afbjorklund** [comment]
I guess this source code never appeared anywhere ? Did some tests with redis, it worked (but slower)

**Comment by vaneyli** [comment]
Sorry to reply so lately. These codes have been admitted to open by special department of my company. It will be seen in the github in short time.
